core/server/master/src/main/java/alluxio/master/FaultTolerantAlluxioMasterProcess.java

```diff
@@ -165,7 +168,9 @@ private boolean gainPrimacy() throws Exception {
     }
     mServingThread = new Thread(() -> {
       try {
-        startServing(" (gained leadership)", " (lost leadership)");
+        // Starts disabled services after gained primacy.
```

Comment:
Remove this comment line because the logic is in `startCommonServices()`. Add comment in `startCommonServices()` if you think it's necessary.

Reply:
Done